bibliopixel/animation/sequence.py

```diff
@@ -11,20 +11,21 @@ def make_animation(a):
                 desc = a
             else:
                 desc = {'animation': a[0], 'run': a[1]}
-            desc = aliases.resolve_aliases(desc)
+            desc = aliases.resolve(desc)
+            # desc['run']['threaded'] = True
```

Comment:
Why??? My change was that this was actually broken!! When you changed the name of resolve_aliases. You never tested sequence when you make that change. By why do you want this to be a feature flag? This is very much required here.